### PR TITLE
feat: add PremiumV2LRS disk type support

### PR DESCRIPTION
**Pull request by andyzhangx**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
feat: add PremiumV2LRS disk type support

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
feat: add PremiumV2LRS disk type support
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
feat: add PremiumV2LRS disk type support
```

### CODE REVIEW

File: pkg/azureclients/vmssclient/interface.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// APIVersion is the API version for VMSS.
-	APIVersion = "2020-12-01"
+	APIVersion = "2022-03-01"
```
Comment (feiskyer) [inline pkg/azureclients/vmssclient/interface.go:31]: do we need to update the apiversion in package name as well, e.g. L23 `"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2021-07-01/compute"` is still using `2021-07-01`.

Comment (andyzhangx) [reply pkg/azureclients/vmssclient/interface.go:31]: I have updated to latest `2021-12-01` package, actually there is no api changes between these two packages for disk functionality, that's why `2021-07-01` also works. Anyway, I have updated to latest package.

**Review by feiskyer — COMMENTED** [review]
/lgtm

**Pull request merged**